NitroxClient/Communication/MultiplayerSession/SessionManagers/DefaultMultiplayerSessionManager.cs

```diff
@@ -9,42 +9,41 @@
 using NitroxModel.MultiplayerSession;
 using NitroxModel.Packets;
 using NitroxModel.Serialization;
+using NitroxModel.Networking;
 
-namespace NitroxClient.Communication.MultiplayerSession
+namespace NitroxClient.Communication.MultiplayerSession.SessionManagers
 {
-    public class MultiplayerSessionManager : IMultiplayerSession, IMultiplayerSessionConnectionContext
+    public class DefaultMultiplayerSessionManager : IMultiplayerSession, IMultiplayerSessionConnectionContext
```

Comment:
Why do you rename this class? You didn't added any alternativs afair.

Reply:
havent added one yet, the P2P will likely require custom ones however

Reply:
The maybe name it "Direct..." instead of "Default...", what do you think?

Reply:
Other things that get added could also not need a new one, though trying to think up a descriptive name is a tad difficult